department-of-veterans-affairs/va.gov-cms
# VACMS-4347: Add default details class to required fieldsets on clp.

## Description
New details with image formatter doesn't add the asterisk to fieldsets with required fields.
This pr adds the missing asterisk hook class to the fieldsets that are required.

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/108211737-1668c800-70fb-11eb-8d15-f005eb12f6b5.png)

## QA steps
As an administrator, go to `/node/add/campaign_landing_page` and visually verify that the first three and the last fieldsets show a required red asterisk.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
